### DISCUSSION TITLE
Demo with consumer devices

### DISCUSSION THREAD

**Discussion by vromaniello** [stella-cv/stella_vslam#253] [Q&A] [answered]
Hi, I am new on OpenVSLAM and I will use it for a research project.
I would like to know if there is the possibility to apply OpenVSLAM to a smartphone or low-cost device.
I found the UVC (web-cam) example [here](https://openvslam-community.readthedocs.io/en/latest/example.html#slam-with-uvc-camera) without .yaml or other configurations in order to execute it as demo.

What are the steps to perform OpenVSLAM to test using the PC web cam or simple mobile device?
For an USB camera is it necessary to install the ROS package as well?

Thank you very much for the support.

**Comment by ymd-stella** [comment, marked as answer]
Use stella_vslam_ros if you want real-time execution. Otherwise, create a video and give it to run_video_slam.
